src/platform/client/client_platform.cpp

```diff
@@ -57,7 +57,7 @@ int mcp::getgid()
     return ::getgid();
 }
 
-std::pair<std::string, std::string> mcp::get_user_password(mp::Terminal*)
```

Comment:
Changing this breaks a reference to it in `client_platform_windows.cpp:76`

Reply:
Yes, the private PR linked above fixes that.